packages/vscode-extension/src/controls/SampleGallery.tsx

```diff
@@ -39,61 +72,54 @@ export default class SampleGallery extends React.Component<any, any> {
           styles={{ root: { overflow: "visible" } }}
           tokens={{ childrenGap: 20 }}
         >
-          <SampleAppCard
```

Comment:
The contents on sample gallery page for vscode extension is not totally the same with CLI/core as you can see the description has more details.

Reply:
is it by design to have two kinds (scaffold from sample & view sample) of description even for the same sample app?

Reply:
Yes, it's by design because we don't have much space to show more detailed info when scaffolding from sample. But for sample gallery, there's enough space and we want the description as detailed as possible. You can also discuss with PM&UX whether we can align the descriptions and just have one version.

Reply:
Thanks for the clarification. I'm not going to change the behavoir if it's by design as it's out of scope of this PR. I will add `shortDescription` and `fullDescription` to `samples-config.json` to mitigate the issue. Does it sound good to you?